tests/integration-tests/tests/iam/test_iam.py

```diff
@@ -363,6 +369,233 @@ def test_iam_resource_prefix(
             _test_iam_resource_in_cluster(cfn_client, iam_client, cluster.name, iam_resource_prefix)
 
 
+def _remove_from_config(config, item_list):
```

Comment:
Seems like you are building some general purpose utilities to update the config files.  Can you add descriptions and explanations for the functions you've created here?  Its a bit hard to follow how they all fit together.

Reply:
Sure.